src/prefect/cli/deploy.py

```diff
@@ -52,7 +52,7 @@
     RRuleSchedule,
 )
 from prefect.client.utilities import inject_client
-from prefect.deployments import find_prefect_directory, register_flow
+from prefect.deployments import register_flow
```

Comment:
Can we get rid of `register_flow` too? It serves no purpose with the ability to deploy via flow name removed.

Reply:
Applied in [dbd41c3](https://github.com/PrefectHQ/prefect/pull/12732/commits/dbd41c39b0ea90121e4458d868e7699c27b063f7), also removing other prefect directory-related functions, if you don't mind to look again since it removes quite a bit.